api/v1beta1/disruption_webhook.go

```diff
@@ -609,8 +617,20 @@ func safetyNetCountNotTooLarge(r *Disruption) (bool, string, error) {
 		userCountVal = float64(userCountInt)
 	}
 
+	logger.Debugw("comparing estimated target count to total existing targets",
+		"disruptionName", r.Name,
+		"disruptionNamespace", r.Namespace,
+		"namespaceThreshold", namespaceThreshold,
+		"clusterThreshold", clusterThreshold,
+		"estimatedEligibleTargetsCount", targetCount,
+		"namespaceCount", namespaceCount,
+		"totalCount", totalCount,
+		"calculatedPercentOfTotal", userCountVal/float64(totalCount),
+		"estimatedTargetCount", userCountVal,
```

Comment:
I was unsure if we should try to make these tags at all intelligible to users who don't have the code in front of them, so I did so. The alternative is to capture every intermediate variable and tag them with their exact names, e.g. `"targetCount", targetCount`